documentation/staging/content/userguide/managing-operators/conversion-webhook.md

```diff
@@ -0,0 +1,112 @@
+To check if the conversion webhook is deployed and running,
+see [Troubleshooting]({{<relref "/userguide/managing-operators/troubleshooting#check-the-conversion-webhook-deployment">}}).
+
+To prevent the removal of conversion webhook during the `helm uninstall` command, set the Helm configuration value `preserveWebhook` to `true` during installation using the `helm install` command. 
+
+The following table describes the behavior of different `Helm` commands (with the Operator Helm chart) and the Helm configuration values:
+
+| Helm command with the operator Helm chart | Helm configuration values | Behavior |
+| --- | --- | --- |
+| Helm install | Default | Operator and conversion webhook installed. |
+| Helm install | Default and existing `weblogic-operator-webhook` deployment | Operator installed. |
+| Helm install | `webhookOnly=true` | Conversion webhook installed. |
+| Helm install | `preserveWebhook=true` | Operator and webhook installed. |
+| Helm uninstall | | Operator and webhook deployment uninstalled. |
+| Helm uninstall with `preserveWebhook=true` set during `helm install` | | Operator deployment uninstalled (and webhook deployment preserved). |
+
+#### Uninstall the conversion webhook
+When you [uninstall the operator]({{<relref "/userguide/managing-operators/installation#uninstall-the-operator">}}) using the `helm uninstall` command, it removes the conversion webhook 
+associated with the release and its resources from the Kubernetes cluster. However, if you **only** installed the conversion webhook using `webhookOnly=true` Helm configuration value, then run the `helm uninstall` 
+command separately to remove the conversion webhook associated with the release and its resources.
+
+For example, assuming the Helm release name for the conversion webhook is `sample-weblogic-conversion-webhook`,
+and the conversion webhook namespace is `sample-weblogic-conversion-webhook-ns`:
+
+```text
+$ helm uninstall sample-weblogic-conversion-webhook -n sample-weblogic-conversion-webhook-ns
+```
+This command deletes the conversion webhook Deployment (`weblogic-operator-webhook`), and it also deletes the conversion resources such as service and Secrets.
+
+{{% notice warning %}}
+The `helm uninstall` command does not delete the conversion definition stored in the Domain CRD. However, the remaining conversion definition in the CRD will cause problems if the conversion webhook runtime is not available. When this happens, you will see a `conversion webhook not found` error when creating a Domain using `weblogic.oracle/v8` schema domain resource. To avoid this error, manually patch the Domain CRD to set the conversion strategy to `None` and remove the `Webhook` details using the below command. Alternatively, use the `preserveWebhook=true` Helm configuration value when installing the operator and conversion webhook using the `helm install` command; this will prevent the removal of the conversion webhook on an subsequent `helm uninstall` of the operator. 
```

Comment:
Maybe start with a sentence that provides high level context, something like:  "You may not want to uninstall a webhook if you have multiple operators deployed to the Kubernetes cluster, or want to continue being able to deploy domains even when an operator isn't installed)."

Reply:
Also, I suspect it'd more clear to put this waarning right at the top of this section (right under "#### Uninstall the conversion webhook").